lib/linguist/languages.yml

```diff
@@ -1866,6 +1865,16 @@ HTML+PHP:
   codemirror_mode: php
   codemirror_mime_type: application/x-httpd-php
   language_id: 151
+HTML+Razor:
+  type: markup
+  tm_scope: text.html.razor
+  group: HTML
+  extensions:
+  - ".cshtml"
+  ace_mode: razor
+  codemirror_mode: htmlmixed
+  codemirror_mime_type: text/html
+  language_id: 432
```

Comment:
🤔 This appears to have been manually set. Please use `script/update-ids` as detailed in CONTRIBUTING.md to set the language ID. You'll need to remove `432` from your local file before running the script in order for it to be replaced.

Reply:
Wow! We should probably detect that in tests.

Reply:
Yup. Coming up.

Reply:
Sorry for that. I was experiencing mysterious issues when trying to follow the manual at CONTRIBUTING.md. `byebug 10.0.2` is throwing exceptions:
```
Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

    current directory: /home/artyom/Github/linguist/vendor/gems/ruby/2.5.0/gems/byebug-10.0.2/ext/byebug
/usr/bin/ruby2.5 -r ./siteconf20181107-23957-1cf8lyo.rb extconf.rb
mkmf.rb can't find header files for ruby at /usr/lib/ruby/include/ruby.h
```

Reply:
@worldbeater You need the ruby dev package, `ruby-dev` under Ubuntu.